docker/Dockerfile.proxy_init_ubuntu

```diff
@@ -0,0 +1,3 @@
+FROM docker.io/centosistio/iptables_ubuntu:20170924-897e396
```

Comment:
why is the ubuntu base image changed ?

Reply:
a base image of iptables from Ubuntu with centos binaries results in 503 errors.  I tracked this down partially to the fact that xtables-multi receives a permission denied (via strace) so the iptables rules are never set.  The root cause is still unclear.

To be more clear, I think the correct answer to this problem is to build all dependency binaries statically (i.e. iptables) and use FROM scratch.  This could result in a functional mesh of ubuntu-built binaries loaded into a scratch container that works on both operating systems.

I'm not sure I have time for the next 3 weeks to investigate a distroless solution in further detail.  Hence I submitted a PR so others interested in CentOS can possibly take this up (or I'll sort it out when I return from travel).

Reply:
thanks for the general explanation to my toplevel question, this inline question though was specifically about why the _ubuntu base image is changing the gcr.io current base to docker.io/centosistio/iptables_ubuntu (ie nothing related to centos)

Reply:
The change of the hub and namespace was simply for my personal testing and should not merge as is.  As I can't push to gcr.io/istio-testing, this would be a regression for a commit to the pilot repository.  I'm not all that sure we really want to take this model forward (unless distroless doesn't work), however, I wanted to provide some context for others that may take up this work in the event that someone wants to solve it now, vs in a few weeks when I return from travel.  The change of the image name to iptables_ubuntu is needed by the patch to work correctly given the explanation to your top level question.